jumpcloud-ADMU/Powershell/Form.ps1

```diff
@@ -385,7 +517,7 @@ $lbNetBios.Content = $NetBiosName
 #AzureADInformation
 $lbAzureAD_Joined.Content = $AzureADStatus
 $lbTenantName.Content = $TenantName
-
+$hostname = hostname
```

Comment:
This was marked resolved but wasn't changed with how we are getting the hostname elsewhere in the script. It should be modified like the suggestion here:
```suggestion
$hostname = $env:computername
```

Reply:
Pushed out a change for this